src/platforms/launchpad-icdi/Makefile.inc

```diff
@@ -9,11 +9,11 @@ CPU_FLAGS = -mcpu=cortex-m4 -mthumb -mfpu=fpv4-sp-d16 -mfloat-abi=hard
 CFLAGS += $(INCLUDES) $(CPU_FLAGS) -DSERIAL_NO=$(SERIAL_NO) -DTARGET_IS_BLIZZARD_RB1 -DLM4F -DPART_TM4C123GH6PM
 CFLAGS += -DDFU_SERIAL_LENGTH=9
 
-LINKER_SCRIPT="platforms/tm4c/tm4c.ld"
+LINKER_SCRIPT="platforms/common/tm4c/tm4c.ld"
```

Comment:
Given the other platforms have had their linker scripts moved into the platform directory, does it not make sense to do that for this one too for consistency?

Reply:
Thanks for the feedback! I was thinking that for the `tm4c` platform there only seems to be one chip, and therefore only one ram and rom size, but out of consistency considerations I also agree it makes sense to place it in the directory of the respective platform. I have implemented this.

Additionally, for consistency, I have also renamed the linker script of the native platform.